src/c4/Timer.cc

```diff
@@ -321,6 +322,57 @@ void Timer::pause(double const pauseSeconds) {
   Ensure(elapsed >= pauseSeconds);
   return;
 }
+
+//---------------------------------------------------------------------------//
+//! Print out a summary timing report for averages across MPI ranks.
+void Timer::printline_mean(std::ostream &out, unsigned const p,
+                           unsigned const w) const {
+  using std::setw;
+  using std::ios;
+
+  unsigned const ranks = rtt_c4::nodes();
+
+  double ni = num_intervals, ni2 = ni * ni;
+  rtt_c4::global_sum(ni);
+  rtt_c4::global_sum(ni2);
+  double mni = ni / ranks;
+
+  double u = sum_user_cpu(), u2 = u * u;
+  rtt_c4::global_sum(u);
+  rtt_c4::global_sum(u2);
+  double mu = u / ranks;
+
+  double s = sum_system_cpu(), s2 = s * s;
+  rtt_c4::global_sum(s);
+  rtt_c4::global_sum(s2);
+  double ms = s / ranks;
+
+  double ww = sum_wall_clock(), ww2 = ww * ww;
+  rtt_c4::global_sum(ww);
+  rtt_c4::global_sum(ww2);
+  double mww = ww / ranks;
+
```

Comment:
Would there be any advantage to packing `{ni, ni2, u, u2, s, s2, ww, ww2}` into an array/vector to limit the function to a single `global_sum`?